components/RectangularButton.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import { Button } from 'react-native-elements';
+import { Text, View } from 'react-native';
+
+export default class RectangularButton extends Component {
```

Comment:
Can you change both the button classes into SFC's? (Stateless functional components)

Reply:
yep totally